src/modules/template/schema.js

```diff
@@ -0,0 +1,42 @@
+const templateModel = require('../../models/template')
+
+const template = templateModel.schema
+
+module.exports = {
+  createTemplate: {
```

Comment:
```suggestion
  create: {
```
Đây cũng là 1 cái t đã thay đổi và muốn áp dụng từ nay về sau, chỉ cần là create, getByID,... thay vì createX, getXByID vì nó được đặt trong 1 ngữ cảnh rất cụ thể, không cần viết thêm cho dài dòng.
Tuy nhiên, khi đề cập đến 1 bộ phận, thì cần viết cụ thể hơn chút. VD: Watch module có update**Target**Data, chứ không thể là updateData được.